gcs/src/main/scala/org/broadinstitute/monster/storage/gcs/GcsApi.scala

```diff
@@ -168,7 +168,42 @@ class GcsApi private[gcs] (runHttp: Request[IO] => Resource[IO, Response[IO]]) {
       }
     }
 
-  // TODO: if the data is greater than n bytes use initResumableUpload and the uploadBytes in multiple chunks
+  def createObject(
+    bucket: String,
+    path: String,
+    contentType: `Content-Type`,
+    expectedSize: Long,
+    expectedMd5: Option[String],
+    data: Stream[IO, Byte]
+  ): IO[Unit] = {
+    if (expectedSize < MaxBytesPerUploadRequest.toLong) {
```

Comment:
Do we ever need `MaxBytesPerUploadRequest` as an int? If not, we can type it as a `Long` at its definition site and avoid running this conversion every time.